### PR TITLE
Math Magicians: tests

### PR DESCRIPTION
**Pull request by erickhansx**
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added thorough tests to compare rendered snapshots of the components.
- [x] I have added tests for the files operate.js and calculate.js mocking user interaction. 
- [x] I have reviewed that there were no linter errors locally.

### CODE REVIEW

File: src/__tests__/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    render(<Calculator />);
+    await userEvent.click(screen.getByRole('button', { name: /2/ }));
+    await userEvent.click(screen.getByRole('button', { name: 'x' }));
+    await userEvent.click(screen.getByRole('button', { name: /7/ }));
+    await userEvent.click(screen.getByRole('button', { name: /=/ }));
+
+    const el = screen.getByTestId('testp').innerHTML;
+    expect(el).toBe('14');
+    expect(el).not.toBe(null);
+  });
```
Comment (tufoinnkuo10) [inline src/__tests__/operate.test.js:16]: - [x] Great job. Please kindly test the other operation methods of `addition,` `division,` and `modulo`

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @erickhansx 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/__tests__/operate.test.js (hunk 1 above)
Comment (erickhansx) [reply src/__tests__/operate.test.js:16]: Thank you for the observation! I just applied the suggested changes. 
PrayingCatGIF

**Review by mwanawabangona — APPROVED** [review]
Hi @erickhansx 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the request change!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**